dts/arm/st/l4/stm32l4p5.dtsi

```diff
@@ -12,6 +12,7 @@
 
 / {
 	sram0: memory@20000000 {
+		compatible = "mmio-sram";
```

Comment:
not required, the sram0: memory node is given by the stm32l4.dtsi
	sram0: memory@20000000 {
		compatible = "mmio-sram";
	};

Reply:
This is overridden by stm32l4p5.dtsi's `sram0` re-declaration. Please check the discussion above